src/shared/lib/validation/opportunity/sprint-with-us.ts

```diff
@@ -76,8 +77,8 @@ export function validatePhaseRequiredCapability(raw: any): Validation<CreateSWUO
 }
 
 export function validateSWUOpportunityInceptionPhase(raw: any, opportunityAssignmentDate: Date): Validation<ValidatedCreateSWUOpportunityPhaseBody, CreateSWUOpportunityPhaseValidationErrors> {
```

Comment:
Are the arguments to these validation functions meant to be the entire SWU opp object? Can't use them in the front-end if that's the case -- much easier if the `raw` value is the thing to be validated. In this case, that would be the `completionDate`. Could you review the other validation functions if that's the case? Let me know if you want to discuss.

Reply:
So that validation function was meant for validating a phase body on the SWU opp create body.  That's the raw in this case.  I pass in the opportunityAssignmentDate because it's needed to validate the start date for the phase.  I might be misunderstanding.